node/src/reactor/main_reactor/tests.rs

```diff
@@ -107,7 +102,7 @@ impl TestChain {
         chainspec.network_config.accounts_config = AccountsConfig::new(accounts, delegators);
 
         // Make the genesis timestamp 60 seconds from now, to allow for all validators to start up.
```

Comment:
```suggestion
        // Make the genesis timestamp 10 seconds from now, to allow for all validators to start up.
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/3396/commits/12a53e511c8a0dc8befb3538bb1a1243b25b5717